bitnami/kibana/templates/configmap.yaml

```
@@ -22,6 +22,9 @@ data:
     {{- if .Values.configuration.server.basePath }}
     server.basePath: {{ .Values.configuration.server.basePath | quote }}
     {{- end }}
+    {{ if .Values.configuration.server.publicBaseUrl }}
+    server.publicBaseUrl: {{ .Values.configuration.server.publicBaseUrl | quote }}
+    {{- end }}
```

I wonder whether it makes sense to set this configuration property based on the ingress hostname (when provided). Sth like this:
```suggestion
    {{- if or .Values.configuration.server.publicBaseUrl .Values.ingress.enabled }}
    server.publicBaseUrl: {{ default .Values.ingress.hostname .Values.configuration.server.publicBaseUrl }}
    {{- end }}
```